dotcom-rendering/src/web/server/index.ts

```diff
@@ -189,12 +190,43 @@ export const renderKeyEvents = (
 	}
 };
 
+export const renderOnwards = (
+	{ body }: { body: CAPIOnwardsType },
+	res: express.Response,
+): void => {
+	try {
+		const {
+			heading,
+			description,
+			url,
+			ophanComponentName,
+			trails,
+			format,
+			isCuratedContent,
+		} = body;
```

Comment:
As a general note here, we use schemas on articles (incl. liveblogs) & fronts to ensure the data passed is correct, but don't do the same for `/blocks`, `/keyevents`, etc - I wonder if we are happy with this?

(If we want to its pretty easy to add new schemas to [get-schema.js](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/scripts/json-schema/get-schema.js), then expand [validate.ts](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/model/validate.ts))

Reply:
I've made a note to look at this. I'm wondering what we want the outcome to be if the schema validation fails. Should that fails the request? I guess it should but it's a bit annoying as ideally, you'd want to know that is going to happen ahead of time but I suppose the separation between repos won't allow this.